modular_skyrat/modules/ices_events/code/ICES_event_config.dm

```diff
@@ -543,11 +543,11 @@
 /**
  * Spess Vines
  *
- * Disabled: Needs rebalancing
+ * 
  */
 /datum/round_event_control/spacevine
-	max_occurrences = 0
-	intensity_restriction = TRUE
```

Comment:
Leave this in so it's capped to once every 45 minutes, not once every 7 minutes.